### PR TITLE
Fix currency display errors in homescreen order activity card

### PR DESCRIPTION
**Pull request by tpaksu**
Fixes https://github.com/Automattic/woocommerce-payments/issues/2100

Add correct currency formatting

Fixes the total amount display on home screen "Orders" activity card. Previously it was using the store's base currency to display all the order totals, now it respects the currency the order was created with. 

Modified the REST API response to contain the formatted currency string, stripped HTML tags and decoded HTML entities before adding it to the API response.

### Detailed test instructions:

-   Create some orders, but in between, change the user currency.
-	Navigate to WooCommerce > Home and if you didn't complete the onboarding setup, complete them to reach the 'Orders/Stock/Reviews' activity cards. 
- 	Check if the order totals show the correct currency symbols.

### CODE REVIEW

**Comment by louwie17** [comment]
@tpaksu thanks for creating a pull request for this, it would indeed be great to get this fixed.
Although I noticed that the only changes present is the changelog addition, did you forget to push up the actual fix by change?

**Comment by tpaksu** [comment]
> @tpaksu thanks for creating a pull request for this, it would indeed be great to get this fixed.
> Although I noticed that the only changes present is the changelog addition, did you forget to push up the actual fix by change?

Hey @louwie17, I thought I pushed the changes but now I realize that husky didn't allow me to do that :) I'll do the necessary cleanup and push it in some minutes. Thanks for the heads up!

File: tests/api/reports-orders.php [hunk 1]
```diff
@@ -125,6 +125,7 @@ public function test_reports_schema() {
 		$this->assertArrayHasKey( 'customer_id', $properties );
 		$this->assertArrayHasKey( 'net_total', $properties );
 		$this->assertArrayHasKey( 'num_items_sold', $properties );
+		$this->assertArrayHasKey( 'total_formatted', $properties );
```
Comment (louwie17) [inline tests/api/reports-orders.php:128]: It might also be worth adding a specific test for the `total_formatted` with an order with a different currency.

**Review by louwie17 — APPROVED** [review]
Thanks @tpaksu for working on this, we appreciate it, I think we could use this in the Analytics tables as well :)
I left one comment for an extra test, but pre-approving, as it tested well and changes look good 👍

File: tests/api/reports-orders.php (hunk 1 above)
Comment (tpaksu) [reply tests/api/reports-orders.php:128]: Sure I'll try to add one! Thanks for the review @louwie17

**Pull request merged**